feature/src/main/java/com/teamdontbe/feature/LoadingActivity.kt

```diff
@@ -1,12 +1,39 @@
 package com.teamdontbe.feature
 
+import android.os.Handler
+import android.view.View
+import android.view.animation.Animation
+import android.view.animation.AnimationUtils
 import com.teamdontbe.core_ui.base.BindingActivity
 import com.teamdontbe.feature.databinding.ActivityLoadingBinding
 import kotlin.random.Random
 
 class LoadingActivity : BindingActivity<ActivityLoadingBinding>(R.layout.activity_loading) {
+
+    lateinit var fadeOutAnim: Animation
+
     override fun initView() {
         initLoadingContent()
+
+        initAnimation()
+    }
+
+    private fun initAnimation() {
+        Handler().postDelayed({
+            fadeOutAnim = AnimationUtils.loadAnimation(this, R.anim.anim_loading_fade_out)
+            fadeOutAnim.setAnimationListener(object : Animation.AnimationListener {
+                override fun onAnimationStart(animation: Animation?) {}
+
+                override fun onAnimationEnd(animation: Animation?) {
+                    // 애니메이션이 끝난 후에 액티비티를 종료
+                    binding.layoutLoading.visibility = View.GONE
+                    finish()
+                }
+
+                override fun onAnimationRepeat(animation: Animation?) {}
+            })
+            binding.layoutLoading.startAnimation(fadeOutAnim)
+        }, 1800)
```

Comment:
```suggestion
private fun initAnimation() {
    lifecycleScope.launch {
        delay(1800)
        fadeOutAnim = AnimationUtils.loadAnimation(this@YourActivity, R.anim.anim_loading_fade_out)
        fadeOutAnim.setAnimationListener(object : Animation.AnimationListener {
            override fun onAnimationStart(animation: Animation?) {}

            override fun onAnimationEnd(animation: Animation?) {
                // 애니메이션이 끝난 후에 액티비티를 종료
                binding.layoutLoading.visibility = View.GONE
                finish()
            }

            override fun onAnimationRepeat(animation: Animation?) {}
        })
        binding.layoutLoading.startAnimation(fadeOutAnim)
    }
}
```
p3 : 스레드-핸들러 방식은 deprecated 되었다고 알고있습니다!
코루틴으 lifecylcescope를 이용해 보는 건 어떨까요?

저도 돌려보지는 않아서 될지는 모르겠습니다ㅎㅎ

Reply:
```
private fun initAnimation() {
    lifecycleScope.launch {
        delay(1800)

        val fadeOutAnim = AnimationUtils.loadAnimation(this@YourActivity, R.anim.anim_loading_fade_out)
        binding.layoutLoading.startAnimation(fadeOutAnim)

        delay(fadeOutAnim.duration)

        binding.layoutLoading.visibility = View.GONE
        finish()
    }
}
```

이러한 방법도 있다 하네요

Reply:
goood

Reply:
굿!!! LGTM